### PR TITLE
MAISTRA-621 fix conflict handling when updating namespace labels in s…

### PR DESCRIPTION
**Pull request by rcernich**
…mmr controller

Signed-off-by: rcernich <rcernich@redhat.com>

### CODE REVIEW

File: pkg/controller/servicemesh/memberroll/controller.go [hunk 1]
```diff
@@ -741,7 +741,7 @@ func (r *namespaceReconciler) reconcileNamespaceInMesh(namespace string) error {
 			err = r.client.Update(context.TODO(), namespaceResource)
 			if err != nil {
 				if errors.IsConflict(err) {
-					namespaceResource := &corev1.Namespace{}
+					namespaceResource = &corev1.Namespace{}
```
Comment (luksa) [inline pkg/controller/servicemesh/memberroll/controller.go:744]: Can probably remove this line altogether.

Comment (rcernich) [reply pkg/controller/servicemesh/memberroll/controller.go:744]: Probably, and that was my gut instinct, but I wasn't sure whether or not the client would clear all the fields.  (I know for SAR in member roll validator, it doesn't clear all the status fields.)

Comment (luksa) [reply pkg/controller/servicemesh/memberroll/controller.go:744]: Ah, ok. Then better not remove it for now.

**Review by luksa — APPROVED** [review]
LGTM

**Pull request merged**